docs/usage.md

```diff
@@ -104,6 +107,28 @@ Sample files that can be used as illustrative references can be found in this re
 | last_name | str | User's last name |
 | email | str | User's email |
 
+## `ib_input_instances.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | The value of the data-content-id |
```

Comment:
This note / description may not be too useful for research users. Perhaps `A unique UUID value that is assigned to the input interactive`